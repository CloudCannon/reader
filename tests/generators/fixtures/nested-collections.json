{
	"collections_config": {
		"data": {
			"path": "tests/generators/fixtures/standard/data"
		},
		"fruit": {
			"path": "tests/generators/fixtures/standard/data/fruit"
		}
	}
}
