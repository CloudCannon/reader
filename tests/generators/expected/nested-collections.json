{
  "collections_config": {
    "data": {
      "path": "tests/generators/fixtures/standard/data"
    },
    "fruit": {
      "path": "tests/generators/fixtures/standard/data/fruit"
    }
  },
  "time": "2000-11-22T00:00:00.000Z",
  "cloudcannon": {
    "name": "cloudcannon-reader",
    "version": "0.0.1"
  },
  "version": "0.0.3",
  "data": {},
  "collections": {
    "fruit": [
      {
        "name": "Apple",
        "rating": 5,
        "path": "tests/generators/fixtures/standard/data/fruit/apple.yml",
        "collection": "fruit",
        "url": ""
      },
      {
        "name": "Banana",
        "rating": 4,
        "path": "tests/generators/fixtures/standard/data/fruit/banana.yml",
        "collection": "fruit",
        "url": ""
      }
    ],
    "data": [
      {
        "0": {
          "name": "Jim",
          "interest": "kayaks"
        },
        "1": {
          "name": "Pam",
          "interest": "art"
        },
        "path": "tests/generators/fixtures/standard/data/authors.csv",
        "collection": "data",
        "url": ""
      }
    ]
  }
}
